triple-store/triple-store-test/src/test/resources/foaf/foaf.sparql

```diff
@@ -0,0 +1,9 @@
+
+# query: selectNickName
+prefix foaf: <http://xmlns.com/foaf/0.1/>
+SELECT * {
+	GRAPH ?graphPersons
+				{ ?x 
+                foaf:mailbox ?mbox ;
+			    foaf:nick ?nick	}
+	}
```

Comment:
Coding style: you use tabs instead of spaces to indent

Reply:
Done